test/libyul/YulOptimizerTest.cpp

```diff
@@ -84,7 +84,7 @@ TestCase::TestResult YulOptimizerTest::run(std::ostream& _stream, std::string co
 	else
 		printedOptimizedObject = optimizedObject->toString();
 
-	// Re-parse new code for compilability
```

Comment:
compilability is intentional, it is about whether something can be compiled, not if it is compatible